cpp/src/gandiva/lru_cache.h

```diff
@@ -67,6 +77,32 @@ class LruCache {
       // insert the new item
       lru_list_.push_front(key);
       map_[key] = std::make_pair(value, lru_list_.begin());
+      cache_size_ += sizeof(key);
+      cache_size_ += sizeof(*value.get());
+    }
+  }
+
+  void insertObject(key_type& key, const value_type value, size_t object_cache_size) {
```

Comment:
Check if the method name should follow the pattern to start with capital letter

Reply:
I checked, and I might be the case, so I capitalized the first letter.